beacon/engine/types.go

```diff
@@ -18,6 +18,7 @@ package engine
 
 import (
 	"fmt"
+	ethparams "github.com/ethereum/go-ethereum/params"
```

Comment:
Please use `"github.com/ethereum/go-ethereum/params"` and also fix the go-imports.